modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -218,6 +214,7 @@
           varname => 'GOVUK_NOTIFY_TEMPLATE_ID',
           value   => $govuk_notify_template_id;
       "${title}-GOVUK_NOTIFY_BASE_URL":
+          ensure  => 'absent',
           varname => 'GOVUK_NOTIFY_BASE_URL',
           value   => $govuk_notify_base_url;
```

Comment:
I reckon we will need to get rid of the `value` if we set it to `absent`.

Reply:
value is a required parameter here so we need to leave it. I did need to change it though as I'd removed the declaration of `$govuk_notify_base_url` so the linter wasn't happy.